install/antennas.csv

```diff
@@ -384,6 +384,7 @@ Trimble Navigation Ltd.,TRM115000.00,1431180098,HANM,0.002,0,0,0,2020-05-22T00:1
 Trimble Navigation Ltd.,TRM115000.00,1431180107,SCTY,0.002,0,0,0,2020-06-23T01:06:32Z,2020-10-27T23:29:07Z
 Trimble Navigation Ltd.,TRM115000.00,1431180119,RLTN,0.002,0,0,0,2020-06-22T22:11:38Z,2020-11-03T00:27:07Z
 Trimble Navigation Ltd.,TRM115000.00,1431180121,RLTN,0.002,0,0,0,2022-02-01T00:46:02Z,2022-07-10T23:59:07Z
+Trimble Navigation Ltd.,TRM115000.00,1431180124,WGTT,0.055,0,0,0,2024-07-16T03:09:56Z,9999-01-01T00:00:00Z
```

Comment:
The height should be `0.0522` as in the field logs. Please check with the RIM team.

Reply:
For everyone - Brendon just mentioned to Aleks and I that there was a mistake in the measurement. The proper height is the standard 0.055, so no changes will be made.